CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # Changelog
 
-## v6.0.0 (06/30/2020)
+## v6.0.1 (08/24/2020)
+
+#### Enhancements:
+
+* Update helm chart name from `pureStorageDriver` to `pure-pso`
```

Comment:
Can we change this to :
Update helm chart name from `pureStorageDriver` to `pure-pso` to meet the rigid helm chart naming convention.